docs/release-notes/spring-2022/README.md

```diff
@@ -11,53 +11,59 @@
   ],
 }
 ---
+<cdr-banner type="info" aria-live="polite"><template #icon-left><icon-information-fill inherit-color /></template><span style="font-weight:500">This is our final major release prior to Cedar's migration to Vue 3</span><template #message-body>Cedar's next release will migrate our component library to Vue 3. As a result, this will be the only Vue 2 release we will support post-migration.</template></cdr-banner>
+
+<br>
 
 <cdr-doc-table-of-contents-shell parentSelector='h2' childSelector='h3'>
 
 ## Update Steps
 
-- Update to the latest version of the Cedar packages:
-
 | package name | version |
 |--------------|---------|
 | `@rei/cedar` | ^12.x.x |
-| `@rei/cdr-tokens` | ^11.x.x |
+| `@rei/cdr-tokens` | ^10.x.x |
 | `@rei/cdr-component-variables` | ^8.x.x |
 
 - If your project depends on any shared component packages (i.e, FEDPACK, FEDCOMP, FEDPAGES), you will want to update those packages to the new version of Cedar before updating your micro-site.
 
-
-## New Features
-
-### New X Icons
-
 ## Bug Fixes
 
+- CdrPopover has been updated to address an issue where the component would emit its `open` event twice in certain circumstances.
 ## Deprecations
 
-### Vue2
-In preparation of our Vue3 update the existing vue2 cedar library is now deprecated. As this is a major update that may impact teams differently we will continue our support of these assets for an additional release.
-Additionally, we plan to archive the vue2 repository as "REI-Cedar-Vue2" and publish make this available vis NPM.
+### Vue 2
 
-## Breaking Changes
+In preparation of our Vue 3 migration we are deprecating the Vue 2 based component library. We will support this final release of the Vue 2 library with basic maintainence and bug fixes for an additional 12 months. Once we release our Vue 3 library, we will re-publish our Vue 2 libray as `@rei/cedar-vue-2`.
```

Comment:
To prepare for our Vue 3 migration, we're deprecating the Vue 2 component library. We'll provide basic maintenance and bug fixes to support this final Vue 2 release for an additional 12 months. Once our Vue 3 library is released, we'll republish our Vue 2 library as`@rei/cedar-vue-2`.